intelmq/bots/parsers/bambenek/parser.py

```diff
@@ -32,32 +33,33 @@ def parse_line(self, line, report):
             self.tempdata.append(line)
 
         else:
-            value = line.split(',')
+            m = re.match(r"(?P<ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}),(?P<description>.*), \
```

Comment:
Does not seem to work with IPv6

Reply:
Could we change `.*` (greedy) to `.*?` (non-greedy) here?

Reply:
Ah, I'm sorry, I didn't consider IPv6. Regex for an IPv6 pattern is out of my scope. 
The greedy to non-greedy works for the description group, but not for the URL group.